src/commands/indexer/index.ts

```diff
@@ -447,7 +462,7 @@ export default class Indexer extends HealthCheck {
                   // A bridge event contains a "TransferERC721" event. Because our process handles a whole event bridge event,
                   // instead of the sub events we have to dedup them. So we make sure that this "TransferERC721" is not part of a bridge event.
                   let isPartOfBridgeTx = false
-                  for (const log of interestingTransaction.allLogs!) {
+                  for (const log of interestingLog.allLogs!) {
```

Comment:
So `interestingLog` is an object that contains all the logs? is `interestingLog` a type we defined or something returned from ethers?

Reply:
I renamed the old `InterestingTransaction` to `InterestingLog` because the main prop in the type is the bloomId.